.changeset/quick-pumas-do.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ag.ds-next/react': patch
+---
+
+Removed the filesize third party dependency
```

Comment:
```suggestion
file-upload: Removed the filesize third party dependency
```

Reply:
Our convention at the moment is to prefix the change with the slug of the component. This makes it easier to generate our release notes, which can be found here https://design-system.agriculture.gov.au/updates